student_class.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
+    @classroom = classroom
+  end
+
```

Comment:
- [ ] Since you are calling the `super` method here and the `parent_permission` method from the parent class is a **_keyword_** argument, you ought to pass it a value of the `parent_permission` parameter that is being created for the `Student` class. This will map the variables properly and the `super` method will use the conditions set in the `initialize` method of the `Person` class to be the same condition as those passed as parameters in the `super` method. Kindly implement this change so that you don't get these errors 💻 🖱️:

![image](https://github.com/DianaBeki/school-library/assets/77106962/4844c022-f46c-4775-a358-9773a21e81fa)

#### Hint;

```ruby
def initialize(id, age, classroom, name = 'Unknown', parent_permission: true)
  super(id, age, name, parent_permission:parent_permission)
  @classroom = classroom
end
```